src/gas-api.js

```diff
@@ -1,210 +1,207 @@
-"use strict";
+'use strict';
 
-function pull(code) {
-  const changed = $('.diff-file:checked').toArray().map(e => e.value);
-  const update_promises = changed.filter(f => code.github[f])
-  .map((file) => {
-    const match = file.match(/(.*?)\.(gs|html)$/);
-    if (!match || !match[1] || !match[2]) {
-      showAlert('Unknow Error', LEVEL_ERROR);
-      return;
-    }
-    const name = match[1];
-    const type = match[2];
+class Gas {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/76f8108a25ffe1858d5d27c3d7dabf83cc8b0291/src/gas-api.js#L3"></span>‘Gas’ is defined but never used.</td>
    </tr>
  </tbody>
</table>